tests/test_water_mask.py

```diff
@@ -78,6 +78,4 @@ def test_create_water_mask(tmp_path, test_data_dir):
     info_from_img = gdal.Info(output_image)
     info_from_txt = open(validation_text).read()
     # The first 4 lines are file paths; they will be different everytime and don't need to be checked.
-    info_from_img = info_from_img.split('\n')[4:]
-    info_from_txt = info_from_txt.split('\n')[4:]
-    assert info_from_img == info_from_txt
+    assert info_from_img.split('\n')[4:] == info_from_txt.split('\n')[4:]
```

Comment:
Was this causing a `mypy` error? Also, this comment seems unnecessary and is also incorrect (I think it should say `The first 3 lines`), should we just remove it?